src/Symfony/Component/Console/Output/StreamOutput.php

```diff
@@ -106,10 +107,18 @@ protected function hasColorSupport()
             return true;
         }
 
-        return 'Hyper' === getenv('TERM_PROGRAM')
+        if ('Hyper' === getenv('TERM_PROGRAM')
+            || false !== getenv('COLORTERM')
             || false !== getenv('ANSICON')
             || 'ON' === getenv('ConEmuANSI')
-            || str_starts_with((string) getenv('TERM'), 'xterm');
+        ) {
+            return true;
+        }
+
+        $term = (string) getenv('TERM');
+
+        return str_ends_with($term, 'screen')
+            || str_ends_with($term, 'xterm');
```

Comment:
why changing from `str_starts_with` to `str_ends_with` ?

Reply:
damn, mistake